docs/data/charts/axis/axis.md

```diff
@@ -141,98 +132,90 @@ If you use composition you can pass those as props to the `<ChartsGrid />` compo
 
 ### Automatic tick position
 
-You can customize the number of ticks with the property `tickNumber`.
+Use the `tickNumber` property to customize the number of ticks.
 
 :::info
-This number is not the exact number of ticks displayed.
-
-Thanks to d3, ticks are placed to be human-readable.
-For example, ticks for time axes will be placed on special values (years, days, half-days, ...).
+This number does _not_ necessarily represent the exact number of ticks that will be displayed.
+This is because D3 automatically places ticks to optimize for human readability, and it will round up or down from the provided `tickNumber` as needed to accomplish this.
 
-If you set `tickNumber=5` but there are only 4 years to display in the axis, the component might choose to render ticks on the 4 years, instead of putting 5 ticks on some months.
+For example, if you set `tickNumber=5` but there are only four years to display on the axis, the component will render four total ticks (one for each year) instead of trying to divide four years into five.
```

Comment:
```suggestion
For example, if you set `tickNumber=5` but there are only four years to display on the axis, the component renders four total ticks (one for each year) instead of trying to divide four years into five.
```